pyodide_build/buildpkg.py

```diff
@@ -58,21 +58,25 @@ def download_and_extract(buildpath, packagedir, pkg, args):
     return srcpath
 
 
-def patch(path, srcpath, pkg, args):
+def patch(path, srcpath, pkg, args, only_extras=False):
     if (srcpath / '.patched').is_file():
-        return
+        if not only_extras:
+            return
+        else:
+            os.remove(srcpath / '.patched')
```

Comment:
This is nice, I used to do this by had to re-build a specific package.

Reply:
I'm a little concerned this will cause packages to always be rebuilt, though...  We should test that...